openshift/templates/marketplace.yaml

```
@@ -385,7 +389,7 @@ parameters:
 - name: KEYCLOAK_URL
   displayName: Keycloak Url
   description: Keycloak Server Url
-  value: ""
+  value: "https://keycloak-digital-marketplace.apps.exp.openshift.cqen.ca"
```

```suggestion
  value: "https://keycloak-$(POD_NAMESPACE).${CLUSTER_DOMAIN}"
```

Il serait préférable de faire le url plus dynamique afin que le développeur puisse le modifier en fonction de son projet.

---

https://github.com/CQEN-QDCE/digital_marketplace/commit/d9eac02dbee1a71cbda60923cf45a9502b52ea26